FrejaEidClient/src/main/java/com/verisec/frejaeid/client/service/AuthenticationService.java

```diff
@@ -101,4 +119,30 @@ private boolean isPollingTimeExpired(long pollingEndTime) {
         return (System.currentTimeMillis() + pollingTimeoutInMilliseconds) < pollingEndTime;
     }
 
+    public static byte[] readAllBytes(InputStream inputStream) throws IOException {
+        final int bufLen = 4 * 0x400; // 4KB
+        byte[] buf = new byte[bufLen];
+        int readLen;
+        IOException exception = null;
+
+        try {
+            try (ByteArrayOutputStream outputStream = new ByteArrayOutputStream()) {
```

Comment:
Seems unnecessary to have 2 try catch blocks here.

Reply:
Done.